crates/mpc-protocols/src/mpc_events.rs

```diff
@@ -0,0 +1,13 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+use sui_types::{
+    base_types::{ObjectID, SuiAddress},
+    id::ID,
+};
+
+// Rust version of the Move sui_system::dwallet::CreatedProofMPCSessionEvent type
+#[derive(Debug, Serialize, Deserialize, Clone, JsonSchema, Eq, PartialEq)]
+pub struct CreatedProofMPCSessionEvent {
+    pub session_id: ID,
+    pub sender: SuiAddress,
```

Comment:
It's a bit nitpicky, but these names could be more consistent. The first includes the type (`_id`), where the second does not (`_address`). Does the company have a style guide on whether or not to include types in names?

Reply:
We don't have such guide AFAIK. What would make sense for you?

Reply:
Something like `session_id` and `sender_address`?

Reply:
I think the names are good, they did `session_id` because that's actually the name of the variable, not to include the type